package.json

```diff
@@ -10,9 +10,7 @@
         "color": "#ffe04b",
         "theme": "light"
     },
-    "activationEvents": [
-        "*"
-    ],
+    "activationEvents": ["*"],
```

Comment:
Please no formatting changes to `package.json`, as `npm` uses `JSON.stringify` and this will cause changes back after running `npm install --save`.

Reply:
@szeck87 We really need to create `JSON.stringify` beautifier to handle these files. So much to do 😫